crucible-llvm/src/Lang/Crucible/LLVM.hs

```diff
@@ -1,5 +1,5 @@
-{-# LANGUAGE EmptyCase #-}
```

Comment:
is possible to configure `hlint` to delete the line (rather than truncate it)?

Reply:
I'm happy to manually fix these, I doubt it's configurable. 

By the way, `hlint` by default only provides warnings, I used the `--refactor` flag to apply a few auto-fixes.